scripts/expo/commands/publishCodeReview.js

```diff
@@ -48,14 +48,16 @@ module.exports = () => {
     }),
   });
 
-  const githubPullRequestId = env.CI_PULL_REQUEST_ID;
```

Comment:
This is not OK - you need to use values from this file: https://github.com/kiwicom/react-native-app/blob/master/scripts/expo/environment.js (`env.*` is not `process.env` here).

Reply:
I know, just trying... 🙂